source/administration-guide/manage/admin/content-flagging.rst

```diff
@@ -0,0 +1,90 @@
+
+By making every team member a first line of defense against sensitive-data exposure, content flagging strengthens mission-critical, secure deployments and supports compliance with organizational and regulatory data-handling standards.
+
+Before you begin
+----------------
+
+You must be a System Admin in Mattermost. You need to identify the individuals or groups to be Content Reviewers, and you need to decide whether flagged messages should be hidden from users in Mattermost while under review.
+
+Enable
+-------
+
+Content flagging isn't enabled by default. To enable content flagging:
+
+1. Go to **System Console > Site Configuration > Content Flagging**. 
+2. Set **Enable content flagging** to **True**.
+
+.. tip::
+
+    Alternatively, you can configure content flagging via the :ref:`config.json file or through environment variables <administration-guide/configure/site-configuration-settings:content flagging>`.
+
+Configure
+---------
+
+1. Under **Content Reviewers**, define who should review flagged content:
+
+   - **Same reviewers for all teams**: Set to **True** to apply one global reviewer list across all teams, or **False** to configure reviewers per team.
+   - **Reviewers**: Start typing to search for users to assign as content reviewers.
+   - **Additional reviewers**: Optionally include:
+
+     - **System Administrators**: System admins receive flagged posts for all teams they are part of.
+     - **Team Administrators**: Team admins receive flagged posts for their respective teams.
+2. Under **Notification Settings**, specify who receives updates at each stage of the flagging workflow when content is flagged or reviewed:
+
+   - **Notify when content is flagged**: Reviewer(s), Author.
+   - **Notify when a reviewer is assigned**: Reviewer(s).
+   - **Notify when content is removed**: Reviewer(s), Author, Reporter.
+   - **Notify on dismissal**: Reviewer(s), Author, Reporter.
+
+   All notifications are sent via the **content-review** bot as direct messages.
+
+3. Under **Additional Settings**, configure how the flagging workflow behaves:
+
+   - **Reasons for flagging**: Define the preset categories that appear in the flagging dialog (for example: **Inappropriate content**, **Sensitive data**, **Security concern**, **Harassment or abuse**, **Spam or phishing**).
+   - **Require reporters to add comment**: Set to **True** to require users to add a short explanation when flagging a message.
+   - **Require reviewers to add comment**: Set to **True** to require reviewers to add a comment when resolving a flag.
+   - **Hide message from channel while it is being reviewed**: Set to **True** to automatically hide flagged messages from the channel until review is complete.
```

Comment:
It would be worth specifying that if a root post of a thread is flagged, the entire thread would be hidden and become inaccessible.